storybooknative/package.json

```diff
@@ -4,7 +4,10 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest",
+    "test-hack-setup": "mv .babelrc .babelrc.bak; mv babel.config.js.bak babel.config.js;",
+    "test-hack-teardown": "mv .babelrc.bak .babelrc; mv babel.config.js babel.config.js.bak;",
+    "test": "yarn test-hack-setup; jest; yarn test-hack-teardown",
+    "test-update": "yarn test-hack-setup; jest -u; yarn test-hack-teardown",
```

Comment:
could all of this be part of a node-script instead, so we don't need to pollute the package.json file?